src/test/ui/rfc-1937-termination-trait/termination-trait-test-wrong-type.stderr

```diff
@@ -1,14 +1,13 @@
-error[E0277]: `main` has invalid return type `f32`
+error[E0277]: the trait bound `f32: Termination` is not satisfied
```

Comment:
This change in error message is technically correct since a unit test is not the program entry `main` but some might consider this a regression in the quality of the error message.